limits.h

```diff
@@ -26,11 +26,14 @@
 // This is defined by the largest homing flag in the system. The Y flag is
 // the largest with a length of just under 40 mm.
 #define MAXFLAGLEN 40.0
+#define MAXSERVODIST 15.0
```

Comment:
We should probably discuss if this is smart.  Full travel is about 11 or 12.  Do we want to be able to do this move from zero?